config/config.yaml

```diff
@@ -10,9 +11,12 @@ kraken:
             -   scenarios/node_scenarios_example.yml
         -   pod_scenarios:
             - -    scenarios/openshift-apiserver.yml
-            - -    scenarios/openshift-kube-apiserver.yml
+             - -    scenarios/openshift-kube-apiserver.yml
         -   time_scenarios:                                # List of chaos time scenarios to load
             - scenarios/time_scenarios_example.yml
+        -   litmus_scenarios:
+            - - https://hub.litmuschaos.io/api/chaos/1.9.1?file=charts/generic/node-cpu-hog/rbac.yaml
```

Comment:
Maybe consume the rbac file local to scenarios dir to be able to modify the namespace? Thoughts?

Reply:
I'm not sure which is better here. Might be good to keep it has the github link to keep it up to date but could see it both ways

Reply:
Sorry for late comment.   GitHub link has disadvantage of not working in disconnected clusters.   Probably not a common scenario and not a blocker for this PR.   But does using the link make the node_hog_rbac.yaml file superfluous?  Or we should keep it around?